### PR TITLE
Run formatter on the app

### PR DESCRIPTION
**Pull request by david-livefront**
## 🎟️ Tracking

N/A

## 📔 Objective

This PR applies the formatter to the app and cleans up a bunch of warnings from Lint.

No functional changes are in this PR.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Review by dseverns-livefront — APPROVED** [review]

**Review by SaintPatrck — DISMISSED** [review]
Left a question for my own knowledge. Looks much better. I actually like how we're handling "nullable boolean Elvis", so I'm glad we can ignore that warning globally.

File: app/build.gradle.kts [hunk 1]
```diff
@@ -135,6 +135,7 @@ android {
         unitTests.isReturnDefaultValues = true
     }
     lint {
+        lintConfig = file("../docs/bitwarden_lint.xml")
```
[the remark this answers is not in the document]
Comment (david-livefront) [reply app/build.gradle.kts:138]: I created this from the default config plus I disabled the `NullableBooleanElvis` check. So I would expect them all to be configured properly.

The docs seem to indicate that it is additive:
>A configuration file that you can use to specify any lint checks that you want to exclude and to customize problem severity levels.

**Review by SaintPatrck — APPROVED** [review]

**Comment by david-livefront** [comment]
Thanks @SaintPatrck & @dseverns-livefront

**Pull request merged**